tests/api_tests/test_dict_score_modifiers.py

```diff
@@ -0,0 +1,300 @@
+import uuid
+
+from tests.marqo_test import MarqoTestCase
+
+
+class TestDictScoreModifiers(MarqoTestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+
+        cls.structured_index_name = "structured_" + str(uuid.uuid4()).replace('-', '')
+        cls.unstructured_index_name = "unstructured_" + str(uuid.uuid4()).replace('-', '')
+
+        cls.create_indexes([
+            {
+                "indexName": cls.structured_index_name,
+                "type": "structured",
+                "vectorNumericType": "float",
+                "model": "open_clip/ViT-B-32/laion2b_s34b_b79k",
+                "normalizeEmbeddings": True,
+                "textPreprocessing": {
+                    "splitLength": 2,
+                    "splitOverlap": 0,
+                    "splitMethod": "sentence",
+                },
+                "imagePreprocessing": {"patchMethod": None},
+                "allFields": [
+                    {"name": "text_field", "type": "text", "features": ["lexical_search"]},
+                    {"name": "double_score_mods", "type": "double", "features": ["score_modifier"]},
+                    {"name": "long_score_mods", "type": "long", "features": ["score_modifier"]},
+                    {"name": "map_score_mods", "type": "map<text, float>", "features": ["score_modifier"]},
+                    {"name": "map_score_mods_int", "type": "map<text, int>", "features": ["score_modifier"]},
+                ],
+                "tensorFields": ["text_field"],
+                "annParameters": {
+                    "spaceType": "prenormalized-angular",
+                    "parameters": {"efConstruction": 512, "m": 16},
+                }
+            },
+            {
+                "indexName": cls.unstructured_index_name,
+                "type": "unstructured",
+                "model": "open_clip/ViT-B-32/laion2b_s34b_b79k"
+            }
+        ])
+
+        cls.indexes_to_delete = [cls.structured_index_name, cls.unstructured_index_name]
```

Comment:
I don't think you need this. Doesn't the base class add the indexes to this list when you call create?

Reply:
From: https://github.com/marqo-ai/marqo-api-tests/blob/mainline/tests/marqo_test.py 

I think this line is necessary